Pytest: "found no collectors", cannot test my changes

When following the astropy contributing and testing guidelines:
[here](https://docs.astropy.org/en/latest/development/testguide.html#testing-guidelines)

I get this error after running `pytest` in the root `astropy` repo.
The error only occurs if I have made any changes to astropy, but tests all pass otherwise.

Any advice would be greatly appreciated.

```(astropy-dev) rithwik@Rithwiks-MacBook-Pro astropy % pytest
============================= test session starts ==============================
platform darwin -- Python 3.13.2, pytest-8.3.5, pluggy-1.5.0

Running tests with Astropy version 7.1.dev876+ge06963e7b0.d20250427.
Running tests in astropy docs.

Date: 2025-04-27T13:15:35

Platform: macOS-10.16-x86_64-i386-64bit-Mach-O

Executable: /Users/rithwik/opt/anaconda3/envs/astropy-dev/bin/python

Full Python Version: 
3.13.2 | packaged by Anaconda, Inc. | (main, Feb  6 2025, 12:54:57) [Clang 14.0.6 ]

encodings: sys: utf-8, locale: UTF-8, filesystem: utf-8
byteorder: little
float info: dig: 15, mant_dig: 15

Package versions: 
Numpy: 2.2.5
Scipy: 1.15.2
Matplotlib: 3.10.1
h5py: 3.13.0
Pandas: 2.2.3
PyERFA: 2.0.1.5
Cython: not available
Scikit-image: not available
asdf-astropy: 0.7.1
pyarrow: 19.0.1

Using Astropy options: remote_data: none.

CI: undefined
ARCH_ON_CI: undefined
IS_CRON: undefined

rootdir: /Users/rithwik/Desktop/astropy
configfile: pyproject.toml
testpaths: astropy, docs
plugins: astropy-0.11.0, remotedata-0.4.1, filter-subpackage-0.2.0, doctestplus-1.4.0, astropy-header-0.2.2, hypothesis-6.131.9, arraydiff-0.6.1, asdf-4.1.0, mock-3.14.0, cov-6.1.1, xdist-3.6.1
collected 0 items / 1 error                                                    

==================================== ERRORS ====================================
______________________________ ERROR collecting . ______________________________
../../opt/anaconda3/envs/astropy-dev/lib/python3.13/site-packages/pluggy/_hooks.py:513: in __call__
    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)
../../opt/anaconda3/envs/astropy-dev/lib/python3.13/site-packages/pluggy/_manager.py:120: in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
../../opt/anaconda3/envs/astropy-dev/lib/python3.13/site-packages/_hypothesis_pytestplugin.py:458: in pytest_ignore_collect
    warnings.warn(
E   UserWarning: Skipping collection of '.hypothesis' directory - this usually means you've explicitly set the `norecursedirs` pytest config option, replacing rather than extending the default ignores.
=========================== short test summary info ============================
ERROR . - UserWarning: Skipping collection of '.hypothesis' directory - this usually ...
=============================== 1 error in 0.64s ===============================
ERROR: found no collectors for /Users/rithwik/Desktop/astropy/astropy

ERROR: found no collectors for /Users/rithwik/Desktop/astropy/docs```

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -206,11 +206,11 @@ testpaths = [
     "docs",
 ]
 norecursedirs = [
+    ".*",
     "docs[\\/]_build",
     "docs[\\/]generated",
     "astropy[\\/]extern",
     "astropy[\\/]_dev",
-    "astropy[\\/]units[\\/]tests[\\/]data",
 ]
 astropy_header = true
 doctest_plus = "enabled"

